### ISSUE TITLE
Change permissions to use --filesystem=home (or host)

### ISSUE THREAD

**Issue by fbugno** [flathub/cc.retroshare.retroshare-gui#2]
By default, bubblewrap restrict access to user home folder, preventing the user to access his files to share.

Sure, there is the option to copy all files that the user want to share from/to .var/app/cc.retroshare.retroshare-gui/.retroshare/, but this is impractical for people with lots of files and not user friendly at all.

The simple solution is to give the application access to user's home folder.

**Comment by TingPing** [comment]
Note that if this is done the data in `~/.retroshare` will effectively be lost (still on disk but in a different location).

**Comment by fbugno** [comment]
That's true. I took a peek at qbittorrent and transmission flatpak manifests, and they also went to grant host filesystem permission, but i'm not aware if they ever had to map their configuration folder somehow.

Do you want to give a try? I will make a pull request with some modifications (.desktop patching, and systray status icon and this permission issue). Feel free to comment on it, accept, reject or modify it.

**Comment by slrslr** [comment]
I have this issue too (Linux Debian). The file browser (Choose a directory to share) does not show the mountpoints in the drive root / even these are set to allow current Linux user to read these. @G10h4ck @Lionir @fbugno @rszibele

**Comment by csoler** [comment]
I'm just trying to figure out where in the .yaml file this can be modified. I suppose it should be possible to add --filesystem=home so that it is included in the generated wrapper.

**Comment by csoler** [comment]
ok got it. The option "--filesystem=host" should be added in the finish-args section.

**Comment by JohannesKleine** [comment]
Problem solved, please close.
